src/Kernel32.Desktop/Kernel32+MODULEENTRY32.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Linq;
+    using System.Runtime.InteropServices;
+    using System.Text;
+
+    /// <content>
+    /// Contains the <see cref="MODULEENTRY32" /> nested struct.
+    /// </content>
+    public partial class Kernel32
+    {
+        /// <summary>
+        /// Describes an entry from a list of the modules belonging to the specified process.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+        [SuppressMessage(
+            "StyleCop.CSharp.MaintainabilityRules",
+            "SA1401:Fields must be private",
+            Justification = "Used in DllImport Marshaling.")]
+        [OfferIntPtrPropertyAccessors]
+        public unsafe partial struct MODULEENTRY32
+        {
+            /// <summary>
+            /// The size of the structure, in bytes. Set automatically by the constructor.
```

Comment:
Rather than "by the constructor" perhaps you should say: `by the <see cref="Create"/> method.`